res/css/structures/_MainSplit.scss

```
@@ -23,15 +23,15 @@ limitations under the License.
 }
 
 .mx_MainSplit > .mx_RightPanel_ResizeWrapper {
-    // no padding on the left. The spacing is taken care of by the main split content.
-    padding: 5px 5px 5px 0px;
-    margin-left: 8px;
+    padding: $container-gap-width;
+    padding-left: calc($container-gap-width/2); // The resizer should be centered: only half of the gap-width is handled by the right panel. The other half by the RoomView.
```

We typically space out the arguments to `calc`:

```suggestion
    padding-left: calc($container-gap-width / 2); // The resizer should be centered: only half of the gap-width is handled by the right panel. The other half by the RoomView.
```